src/components/document/resize-panel-divider.scss

```diff
@@ -0,0 +1,40 @@
+@import "../vars.sass";
+
+  .resize-panel-divider {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    position: absolute;
+    top: $header-height;
+    height: calc(100vh - (#{$header-height} + #{$workspace-content-margin} * 2));
```

Comment:
Are all of these heights determined with these complex equations?  Just curious.

Reply:
All the workspace heights are calculated that way.